### ISSUE TITLE
Question about the Learning rate of the BotRGCN

### ISSUE THREAD

**Issue by 13061051** [LuoUndergradXJTU/TwiBot-22#18]
hello, I have a simple question. What are the parameters for your BotRGCN. I found in your project https://github.com/BunsenFeng/BotRGCN, the learning rate is set to 0.001, but in this project, it is set to 0.01. which is the real parameter used in your paper, thank you.

**Comment by BunsenFeng** [comment]
@whr000001 @leopoldwhite I don't know which one of you should answer this, but please help!

**Comment by whr000001** [comment]
Hi, thank you for your interest in our work. The value of the learning rate is related to the batch size, generally, a larger batch size means a larger learning rate.

 In the code 'src/BotRGCN/twibot_22/train.py', we train BotRGCN without minibatch, so we choose the learning rate as 0.01. 

You can also use the code 'src/GCN_GAT/train.py' to train BotRGCN with minibatch. You can run the following command to train it:

`
python train.py --dataset Twibot-20
`

The hyperparameters on dataset Cresci-15 and TwiBot-20 are default (learning rate 1e-4 and batch size 128).  On dataset TwiBot-22, we choose the learning rate as 1e-3 and the batch size as 1024, namely, you can run:

`
python train.py --dataset Twibot-22 --batch_size 1024 --lr 1e-3
`

**Comment by 13061051** [comment]
Ok, Thank you!
